lib/charms/layer/jenkins/configuration.py

```diff
@@ -11,8 +12,47 @@
 class Configuration(object):
     """Manage global Jenkins configuration."""
 
+    @staticmethod
```

Comment:
There's no need for the @staticmethod decorator, please just add "self" to the signature and make it a regular instance method.